framework/src/loops/ComputeUserObjectsThread.C

```diff
@@ -191,6 +198,51 @@ ComputeUserObjectsThread::onInternalSide(const Elem * elem, unsigned int side)
   }
 }
 
+void
+ComputeUserObjectsThread::onInterface(const Elem * elem, unsigned int side, BoundaryID bnd_id)
+{
+
+  // Pointer to the neighbor we are currently working on.
+  const Elem * neighbor = elem->neighbor_ptr(side);
+
+  // Get the global id of the element and the neighbor
+  const dof_id_type elem_id = elem->id(), neighbor_id = neighbor->id();
+
+  if (!_interface_user_objects.hasActiveBoundaryObjects(bnd_id, _tid))
+  {
```

Comment:
Don't need `{}`

Reply:
I fixed all of these kind of styles mistake